test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -2,99 +2,103 @@
 
 class ApiUsers::PermissionsControllerTest < ActionController::TestCase
   context "#edit" do
-    should "prevent unauthenticated users" do
```

Comment:
From the commit: "This doesn't change the implementation of the tests yet" -> are there likely to be more changes to the `#edit` tests later? It looks like these are fairly small things for now (and two of the commits are factory changes)

Reply:
Yeah I think I thought there'd be more, but there was only one small thing. I've updated the commit message now.

Reply:
I think we could still do the stuff like relying less on names, but that can come later

Reply:
Oh yeah - not sure what I was seeing yesterday!